src/telemetry/metrics/tls_config_reload.rs

```diff
@@ -54,12 +60,14 @@ impl TlsConfigReload {
 
 impl fmt::Display for TlsConfigReload {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        if !self.by_status.scopes.is_empty() {
+        let inner = self.0.read().expect("lock");
```

Comment:
Nit: IMO, `.expect("lock")` is un-descriptive enough that it may as well just be `unwrap`...

Reply:
Agreed.

On a tangent, it kind of seems like we wish `Mutex` and `RwLock` didn't poison, it might be worth a new PR at some point cleaning up all the `expect`s on our locks somehow...

Reply:
I've avoided this altogether... will post momentarily